_user/Makefile

```diff
@@ -0,0 +1,28 @@
+#
+# Makefile for user app
+#
+# Copyright 2021 Phoenix Systems
+#
+# %LICENSE%
+#
+
+include ../phoenix-rtos-build/Makefile.common
+
+CFLAGS += $(BOARD_CONFIG)
```

Comment:
board config should not be needed for userspace apps

Reply:
Marek, it seems that you read in my minds. The only difference is that I recommend to use `b_build_target` function instead of defining new build function.

Reply:
My reasoning was that `b_build_project` is already invoked in generic `build.project`:

https://github.com/phoenix-rtos/phoenix-rtos-project/blob/248c62ecec7467c849f91e2b2204b6de54638193/build.project#L62-L65

And it's easy to overwrite/undefine full function in custom project not to build sample binaries (just define Your own in `_projects/build.project.ia32-generic-myproject`).

If the build would be part of `b_build_target` it would probably be harder to disable them in custom projects :).